_layouts/home.html

```diff
@@ -12,8 +12,15 @@ <h3 class="archive__subtitle">{{ site.data.ui-text[site.locale].recent_posts | d
   {% assign posts = site.posts %}
 {% endif %}
 
+{% if page.entries_layout == "grid" %}
+<p></p>
```

Comment:
Probably there's a better way to add some padding / space but I've not done front-end / css in years :P 

Please if you like this PR but think there's a better way to add this space, don't ask me to fix this line 😳

Reply:
Yeah definitely not a fan of the empty `<p></p>` element. Styling should be achieved with CSS, not forcing in HTML elements for presentation purposes, it's bad for semantics.

As @iBug suggested, there are enough class name hooks that I'm sure a simple line of CSS can add some margin to increase the white space.

Reply:
Thanks for the tip. I added 0.5em instead to minimize the differences with previous versions (i.e. with lists) while keeping a small visual space between the horizontal line and the posts.

Hey mom see, I'm doing frontend again! :D